src/plugins/intel_cpu/src/emitters/snippets/aarch64/cpu_generator.cpp

```diff
@@ -239,6 +248,7 @@ CPUTargetMachine::CPUTargetMachine(dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
 
     // memory access
     jitters[snippets::op::Load::get_type_info_static()] = CREATE_SNIPPETS_EMITTER(jit_load_memory_emitter);
+    jitters[snippets::op::LoadReorder::get_type_info_static()] = CREATE_SNIPPETS_EMITTER(jit_load_memory_emitter);
```

Comment:
`LoadReorder` is part of [decomposed](https://github.com/openvinotoolkit/openvino/blob/05aba449656e633f68d243a641dbfa03c279e7ef/src/common/snippets/src/pass/transpose_decomposition.cpp#L50) `Transpose` op.

If we want to support `LoadReorder` in this PR, we need to add tests. Please take a look at this GFI: https://github.com/openvinotoolkit/openvino/issues/28488. And fully complete this task please.

Otherwise, let's remove `LoadReorder` from this PR and support decomposed `Transpose` in the separate PR as dev PR for https://github.com/openvinotoolkit/openvino/issues/28488

Reply:
Enabled LoadReorder tests